$NetBSD: patch-configure,v 1.4 2023/06/16 20:59:44 wiz Exp $

Add frameworks required to link under OS X.
Use absolute paths for dlopen(); don't expect users set LD_LIBRARY_PATH.

--- configure.orig	2023-02-07 00:43:37.000000000 +0000
+++ configure
@@ -13375,6 +13375,7 @@ printf %s "checking for ImageIO support.
 printf "%s\n" "$enable_imageio" >&6; }
         if test x$enable_imageio = xyes; then
             IMG_LIBS="-Wl,-framework,ApplicationServices -lobjc $IMG_LIBS"
+            IMG_LIBS="-Wl,-framework,Cocoa -Wl,-framework,Carbon -Wl,-framework,IOKit -Wl,-framework,ForceFeedback $IMG_LIBS"
         else
             CFLAGS="$CFLAGS -DSDL_IMAGE_USE_COMMON_BACKEND"
         fi
@@ -13426,7 +13427,7 @@ find_lib()
         host_lib_path="$ac_default_prefix/$base_libdir $ac_default_prefix/$base_bindir /usr/$base_libdir /usr/local/$base_libdir"
     fi
     for path in $env_lib_path $gcc_bin_path $gcc_lib_path $host_lib_path; do
-        lib=`ls -- $path/$1 2>/dev/null | sed -e '/\.so\..*\./d' -e 's,.*/,,' | sort | tail -1`
+        lib=`ls -- $path/$1 2>/dev/null | sed -e '/\.so\..*\./d' | sort | tail -1`
         if test x$lib != x; then
             echo $lib
             return
